RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -1,14 +1,15 @@
 script "auto_combat.ash"
 
+//since combat functions will never be used outside of combat file, they get declared here instead of the general header file.
 monster ocrs_helper(string page);
 void awol_helper(string page);
-
 boolean registerCombat(string action);
 boolean registerCombat(skill sk);
 boolean registerCombat(item it);
 boolean containsCombat(string action);
 boolean containsCombat(skill sk);
 boolean containsCombat(item it);
+boolean enemyCanBlocksSkills();
```

Comment:
this isn't necessary. Your function is defined before it's called. Those other ones are at the end of the file for some reason hence they're added there.

Reply:
change made